src/streamlink/plugins/cbsnews.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import print_function
```

Comment:
There is no `print` call in the module, so the `__future__` import is unnecessary.